mne/dipole.py

```diff
@@ -268,6 +268,27 @@ def plot_locations(self, trans, subject, subjects_dir=None,
             show_all, ax, block, show, scale=scale, color=color, fig=fig,
             title=title)
 
+    @verbose
+    def to_mni(self, subject, trans, subjects_dir=None,
+               verbose=None):
+        """Convert dipole location from head coordinate system to MNI coordinates.
+
+        Parameters
+        ----------
+        %(subject)s
+        %(trans)s
```

Comment:
```suggestion
        %(trans_not_none)s
```

look at the text on mne/utils/docs.py
to see what it will be replaced with.

Reply:
Alright, got it!